TestownikConsoleApp/Menu.cs

```diff
@@ -4,53 +4,68 @@
 
 namespace TestownikConsoleApp
 {
-    class Menu
+    class MenuClass
     {
-        public void menu()
+        public void Menu()
         {
 
             Console.WriteLine("1 - Losuj");
             Console.WriteLine("2 - Dodaj Pytanie");
             Console.WriteLine("3 - Wyjdź");
-            int caseSwitch = Convert.ToInt32(Console.ReadLine());
             GetPathClass getPath = new GetPathClass();
-            string path = getPath.getrandomfile("Pytania");
+            string path = getPath.GetRandomFile("Pytania");
 
-            try
+            var caseSwitch = Console.ReadLine();
+            int number;
+
+            bool success = Int32.TryParse(caseSwitch, out number);
```

Comment:
jak zrobisz case default jak niżej ci napisałem to wtedy ta zmienna success ci nie będzie potrzebna, tylko samo wywołanie Int32.TryParse. Jak nie uda mi się sparsować tego to dostaniesz 0.